sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIOReadTest.java

```diff
@@ -167,6 +198,21 @@ public void testBuildQueryBasedSource() {
     checkReadQueryObject(read, "foo_query");
   }
 
+  private static SerializableFunction<SchemaAndRowProto, SchemaAndRowProto> rowProtoIdentityFn =
```

Comment:
You can use SerializableFunctions.identity() - or, at least, use a lambda :)

Reply:
I was not able to make the test code run using lambdas without also specifying a coder -- even declaring e.g.

private static SerializableFunction<SchemaAndRowProto, SchemaAndRowProto> identity =
        (input) -> input;

... would still fail when trying to infer the coder type. I'll try SerializableFunction.identity().

Reply:
Done.